django/contrib/staticfiles/storage.py

```diff
@@ -410,7 +410,7 @@ def path_level(name):
                 yield name, hashed_name, processed, substitutions
 
     def clean_name(self, name):
-        return name.replace("\\", "/")
+        return name.strip().replace("\\", "/").removeprefix("/")
```

Comment:
I see you've pushed the `.strip()` down from `stored_name()`. Are there tests that cover it?

Reply:
It's not broken any existing, which is something, but no I've not added any additional ones. I'll have a play with existing tests for it and see if I can exercise it a bit more.